wework/src/styles/globals.css

```
@@ -3,17 +3,59 @@
   --color-border: 224 224 224;
   --color-muted: 245 245 245;
+  --color-sidebar: 229 229 231 / 0.72;
+  --color-sidebar-active: 222 223 226;
+  --color-sidebar-hover: 255 255 255 / 0.7;
+  --color-sidebar-text-primary: 36 40 45;
+  --color-sidebar-text-secondary: 82 89 98;
+  --color-sidebar-text-muted: 136 143 152;
+  --color-mobile-drawer: 238 242 247;
+  --color-popover: 255 255 255;
+  --color-code-bg: 243 244 246;
+  --font-ui: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'PingFang SC', 'Microsoft YaHei', sans-serif;
```

Stylelint (17.12.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)